src/core/lv_obj_pos.c

```diff
@@ -627,6 +627,18 @@ void lv_obj_refr_pos(lv_obj_t * obj)
     if(LV_COORD_IS_PCT(x)) x = (pw * LV_COORD_GET_PCT(x)) / 100;
     if(LV_COORD_IS_PCT(y)) y = (ph * LV_COORD_GET_PCT(y)) / 100;
 
+    /*Handle positionings*/
+    if(pw) {
```

Comment:
Why not fold the two ifs? I.e. `if (pw && LV_COORD_IS_PCT(x) && LV_COORD_GET_PCT(x))...`?

Reply:
Simply due to debugging purposes. It gets optimized out during compilation anyways

Reply:
Hmm, it would surprise me if the compiler can remove the statement, given that `pw` not known during compile time. Or what exactly do you mean by "optimized out"?

Reply:
It is going to group the ifs accordingly and is going to use the least amount of branches possible. Basically it comes down to coding style